### ISSUE TITLE
Recurrent viral pneumonia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6247]
**Preferred term label:**
Recurrent viral pneumonia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased susceptibility to viral pneumonia, as manifested by recurrent episodes of viral pneumonias.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/617241

PMID: 27427983   2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
